src/events/ResultPreviewsManagerEvents.ts

```diff
@@ -17,9 +18,9 @@ export interface IUpdateResultPreviewsManagerOptionsEventArgs {
  */
 export interface IPopulateSearchResultPreviewsEventArgs {
   /**
-   * The text to look up search result previews for.
+   * The suggestion to look up search result previews for.
    */
-  suggestionText: string;
+  suggestion: Suggestion;
```

Comment:
So this would normally be a big `no no` since you are breaking backward compatibility on a publicly documented interface.

But !, since the feature is still pilot and not widely used yet, then I guess it's fine.

Reply:
I didn't think about the fact that events are publicly documented. Should I keep the original parameter there as well for backwards compatibility, then?

Reply:
Since it's a pilot feature, it's okay to change it.